spec/javascripts/index.js

```diff
@@ -1,9 +1,9 @@
 require('libs');
 require('./extra/jasmine-sinon');
 require('./extra/jasmine-jquery');
-
+require('jasmine-ajax');
```

Comment:
Is it possible to use sinon fakeServer here instead of adding this new dependency?